V2.16-RUN-11951-Cluster-visibility (#615)
run-ai/docs: docs/admin/runai-setup/cluster-setup/cluster-install.md

Below are instructions on how to install a Run:ai cluster.

## Prerequisites
Before installing, please review the installation prerequisites here: [Run:ai GPU Cluster Prerequisites](cluster-prerequisites.md).

!!! Important

>     We strongly recommend running the Run:ai [pre-install script](cluster-prerequisites.md#pre-install-script) to verify that all prerequisites are met.

Author: ```suggestion
    It is strongly recommend that you run the Run:ai [pre-install script](cluster-prerequisites.md#pre-install-script) to verify that all prerequisites are met.
```

* (SaaS and remote self-hosted cluster only) Install a trusted certificate to the domain entered above.
*  Run the [Helm](https://helm.sh/docs/intro/install/) command provided in the wizard.

> ## Cluster Table

Reviewer: @jasonnovichRunAI I think we should set the new section after "Validate the installation" and not before

After you have installed your cluster on the platform, you will see it appear in the *Cluster Table*. The *Cluster Table* provides a quick and easy way to see the status of your cluster.

In the left menu, press *Clusters* to view the cluster table. Use *Add filter* to add one or more filter results based on the columns that are in the table. In the *Contains* pane, you can use partial or complete text. Filtered text is ***not*** case sensitive. To remove the filter, press *X* next to the filter.

The table provides the following columns:

* **Cluster**&mdash;the name of the cluster.
* **Status**&mdash;the status of the cluster. For more information see [Cluster status](#cluster-status). Hover over the information icon to see a short description and links to troubleshooting.

> * **Creattion time**&mdash;the timestamp the cluster was created.

Reviewer: ```suggestion
* **Creation time**&mdash;the timestamp the cluster was created.
```

* **URL**&mdash;the URL that was given to the cluster at the time of creation.

> * **Run:ai cluster version**&mdash;the version of the cluster.

Reviewer: @jasonnovichRunAI 
should be "the Run:ai version installed on the cluster"

* **Run:ai cluster UUI**&mdash;the unique ID of the cluster.

### Cluster Status

The following table describes the different statuses that a cluster could be in.

| Status | Description |
| -- | -- |

> | Waiting to connect | The cluster is waiting to connect or has never been connected. |

Reviewer: ```suggestion
| Waiting to connect | The cluster has never been connected. |
```

> | Disconnected | There is no communication from the cluster to the Control Plane. This may be a network issue outside of the Run:ai platform. |

Reviewer: @jasonnovichRunAI 
Let's write something like "This may be due to a network issue" instead of "This may be a network issue outside of the Run:ai platform."

> | Disconnected | All the services that communicate with the control plane are not reporting. This may bean issue outside of of the Run:ai platform. |

Reviewer: ```suggestion
| Disconnected | There is no communication from the cluster to the Control Plane. This may be a network issue outside of the Run:ai platform. |
```

| Missing prerequisites | At least one of the [Mandatory Prerequisites](cluster-prerequisites.md#prerequisites-in-a-nutshell) has not been met. |

> | Service issues | At least one of the *Services* is not working properly. |

Reviewer: ```suggestion
| Service issues | At least one of the *Services* is not working properly. You can view the list of nonfunctioning services for more information |
```

| Connected | All services are connected and up and running. |

> See the [Troubleshooting guide](../../troubleshooting/cluster-health-check.md#verifying-cluster-health) to help troubleshoot statuses.

Reviewer: ```suggestion
See the [Troubleshooting guide](../../troubleshooting/cluster-health-check.md#verifying-cluster-health) to help troubleshoot issues in the cluster.
```
